src/tyr/serializers.cc

```diff
@@ -157,6 +158,18 @@ void openlr(const valhalla::Api& api, int route_index, rapidjson::writer_wrapper
   writer.end_array();
 }
 
+void serializeWarnings(const valhalla::Api& api, rapidjson::writer_wrapper_t& writer){
```

Comment:
so, `warnings` could (and should eventually) be added for all actions/endpoints. and the problem there is that some (or even all) other actions like isochrones/locate etc use another syntax/model for JSON. the one you see above is based on rapidjson. there's another one in baldr/json.h, which is a homegrown JSON builder.

so to support all endpoints, we'll probably have to overload this function. at which point I'm not sure it has such merit to even have it centrally in `serializers.cc`, but let's go with that for now.

Reply:
done..